src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public void getRandomBall() {
```

Comment:
this method should return only one ball and doesn't containt method System.out.println